projects/unit/configs/all_8_datasets/shared_dec.yaml

```diff
@@ -0,0 +1,105 @@
+includes:
+- ../vqa2_dataset_cfg.yaml
+- ../visual_entailment_dataset_cfg.yaml
+
+model_config:
+  unit:
+    base_args:
+      num_queries:
+        detection:
+          detection_coco: 100
+          detection_visual_genome: 100
+        vl:
+          vqa2: 25
+          # hateful_memes: 25
+          visual_entailment: 25
+        glue:
+          glue_qnli: 25
+          glue_qqp: 25
+          glue_sst2: 25
+          glue_mnli_mismatched: 25
+      share_decoders: true
+      decoder_hidden_dim: 768
+      dilation: true
+      use_task_embedding_in_img_encoder: true
+      use_task_embedding_in_lang_encoder: true
+    bert_config:
```

Comment:
Can we add `bert_config` to the base defaults config as well?

Reply:
(moved `bert_config` to `base_args` and changed `detr_model` to `unit_base_model` with format_state_key)